src/setup/kmd/constants.rs

```diff
@@ -0,0 +1,16 @@
+//! Useful setup constants.
+
+use tokio::time::Duration;
+
+/// Directory of the kmd instance.
+/// This directory is generated automatically within the node's directory when the node is created.
+pub const KMD_DIR: &str = "kmd-v0.5";
```

Comment:
I think some file systems don't play nice with `-`. Can this be changed to `_` ?

Reply:
Unfortunately, it's just how the algod dir structure looks like:
```sh
total 1432
-rw-rw-rw-   1    115K Nov 24 10:13 agreement.cdv
-rw-------   1      0B Nov 24 10:12 algod-err.log
-rw-------   1    648B Nov 24 10:13 algod-out.log
-rw-r--r--   1     64B Nov 24 10:12 algod.admin.token
-rw-------   1      0B Nov 24 10:12 algod.lock
-rw-r--r--   1     64B Nov 24 10:12 algod.token
-rw-r--r--   1    205B Nov 24 10:13 config.json
-rw-r--r--   1     94K Nov 24 10:12 consensus.json
-rw-r--r--   1    1.8K Nov 24 10:12 genesis.json
drwxr-xr-x   3     96B Dec 13 09:50 goal.cache
drwxr-xr-x   9    288B Dec 13 09:51 kmd-v0.5
-rw-r--r--   1    152B Nov 24 10:13 logging.config
-rw-r--r--   1    426K Dec 13 09:51 node.log
drwxr-xr-x  17    544B Dec 13 09:51 private-v1
```

we can't do much about it

Reply:
Not every path should be canonicalized as it's often case to use files from some relative paths that we know they'd work just fine.
The case here was only for @lwojtow notice that some filesystems can have problems with such kind of names. Canonicalizing it should solve potential problems as implementation is platform-dependent and should include handling cases difficult for that platform.

Reply:
> Canonicalizing it should solve potential problems as implementation is platform-dependent and should include handling cases difficult for that platform.
 
 Let's skip this improvement for now since most filesystems should work well with `-` (we'll test it on Linux/Mac anyway and if something breaks, we'll make a fix)
---
extra info:
from @asmie:
> ISO9660 would have problem with -

and that means: ISO 9660 (also known as [ECMA](https://en.wikipedia.org/wiki/Ecma_International)-119) is a [file system](https://en.wikipedia.org/wiki/File_system) for [optical disc](https://en.wikipedia.org/wiki/CD-ROM) media.

hopefully no one will run our test suite from a CD, although it would be fun haha